tests/foreman/api/test_errata.py

```diff
@@ -522,7 +522,7 @@ def test_positive_get_count_for_host(self):
             'content-view-id': content_view.id,
             'lifecycle-environment-id': env.id,
             'activationkey-id': activation_key.id,
-        }, force_use_cdn=True)
+        }, force_manifest_upload=True)
```

Comment:
That's a risky change in these specific tests - custom repos may change their contents every week with each new snap, new errata may be added. Our tests rely on exact errata counts in their assertions, meaning if any new errata added - they will fail. Such logic was pretty much ok for 'stable' cdn repos but is risky for custom repos where new errata may be added at any moment. 2 possible actions here:
- Use `force_manifest_upload=True` but update assertions logic to smth like `assertGreaterEqual(errata_counts, 1)`
- Keep `force_use_cdn=True` and update enabling repos with `force=True`

Reply:
Btw if my memory doesn't betray me, at least some of these tests were not forcing cdn before, that `force_use_cdn=True` was introduced because of such failures 🙂

Reply:
@abalakh, I am aware about the issue you mentioned. That repo is for installing katello-agent only (The same way we are using sat-tools 7) and the errata are for packages installed from other repos, we force upload manifest as needed by other RH repos enabled later, so there is no worries about that.

Reply:
Errata is just metadata for package versions, and since both repos are in the same org - if sattools repo has any new packages there will be new applicable errata for that host.
But i can see your point - most errata from sattools shouldn't be applicable for host as we don't really use anything except katello-agent, and katello-agent won't have applicable errata as we install the latest version, so probability of failure because of new errata introduced is quite low in these tests. Ok, consider my comment as a nitpick, not blocker.